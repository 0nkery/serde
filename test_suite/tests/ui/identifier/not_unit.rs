#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
#[serde(field_identifier)]
enum F {
    A,
    #[serde(other)]
    Other(u8, u8),
}

fn main() {}
