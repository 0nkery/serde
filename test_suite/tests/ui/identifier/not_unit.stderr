error: #[serde(other)] must be on a unit variant
 --> $DIR/not_unit.rs:4:10
  |
4 | #[derive(Deserialize)]
  |          ^^^^^^^^^^^

error: aborting due to previous error

